src/commands/deploy.js

```diff
@@ -54,15 +54,17 @@ const generateGCEResources = async (mashrConfigObj) => {
   const dockerfile = await readFile(`${__dirname}/../../templates/docker/Dockerfile`);
   const gemInstallationScript = createGemInstallationScript(mashrConfigObj.mashr.embulk_gems);
   const keyfile = await readFile(`${mashrConfigObj.mashr.json_keyfile}`);
-  const crontab = createCrontab(mashrConfigObj.mashr.embulk_run_command);
+  const embulkScript = createEmbulkScript(mashrConfigObj.mashr.embulk_run_command);
```

Comment:
I like the idea of calling a script from crontab rather than trying to use a command inside of crontab. It seems like for our purpose we benefit from having a script file, where we can do multiple things like create the date variable and run a command.

Reply:
This was from your research :) I made it call a script to be able to set the environment variable to use within `embulk_config.yml.liquid`.